tests/Makefile

```
@@ -193,6 +193,10 @@ $(TOP)/tools/mtouch/SdkVersions.cs: $(TOP)/tools/common/SdkVersions.cs.in
 	@touch $@
 
 xharness/xharness.exe: $(wildcard xharness/*.cs) xharness/xharness.csproj $(TOP)/tools/mtouch/SdkVersions.cs test.config test-system.config .stamp-src-project-files
+	make -j8 -C $(TOP)/external/mono/mcs/class -i do-test PROFILE=MONOTOUCH # TODO: This should not be needed and we should get the dlls from the SDK.
+	make -j8 -C $(TOP)/external/mono/mcs/class -i do-xunit-test PROFILE=MONOTOUCH # TODO: This should not be needed and we should get the dlls from the SDK.
```

```suggestion
	make -j8 -C $(TOP)/external/mono/runtime -i mcs-do-xunit-test-profiles test_profiles="monotouch" # TODO: This should not be needed and we should get the dlls from the SDK.
```

You could even merge it with the above command with 
```
make -j8 -C $(TOP)/external/mono/runtime -i mcs-do-xunit-test-profiles mcs-do-test-profiles test_profiles="monotouch" # TODO: This should not be needed and we should get the dlls from the SDK.
```